ma-copy-artist-album-to-clipboard.user.js

```diff
@@ -9,27 +9,48 @@
 // @copyright  2019+, Rowolta
 // ==/UserScript==
 
-var $, jQuery, artist,
-    tmpl = '<a class="iconContainer ui-state-default ui-corner-all" href="#" title="Copy to clipboard">\
+var artist,
+    tmpl = `<div style="color:#FFF;background: #6D4C4C; padding: 3px; position: absolute; top: -20px; left: -20px; white-space: nowrap"></div>
+            <a class="iconContainer ui-state-default ui-corner-all" href="#" title="Copy to clipboard">\
                 <span class="ui-icon ui-icon-copy">Copy</span>\
-            </a>';
-function renderButtons()
-{
-    $("#band_disco a.other, #band_disco a.album,  #band_disco a.single,  #band_disco a.demo").each(function() {
-        var cpStr = artist + " - " + $( this ).text();
-        $(tmpl).insertBefore($( this )).click(function(event) {
-            event.preventDefault();
-            GM_setClipboard(cpStr + "\n");
-            $('<div style="color:#FFF;background: #6D4C4C; padding: 3px; position: absolute; top: -20px; left: -20px; white-space: nowrap"></div>')
-            .appendTo($(this).css('position', 'relative'))
-                .text('Copied "' + cpStr + '" to clipboard').show().delay(2000).fadeOut();
+            </a>`;
+    tmpl = tmpl.trim();
+function renderButtons() {
+    artist = document.querySelector('h1.band_name a').textContent;
+    let band_disco = document.querySelectorAll("#band_disco a.other, #band_disco a.album,  #band_disco a.single,  #band_disco a.demo");
+    band_disco.forEach(link => {
+        let cpStr = `${artist} " - " ${link.textContent}`;
+        let elements = generateTemplate(tmpl, cpStr);
+        elements.forEach(elem => {
+            link.prepend(elem);
         });
     });
 }
 
-function waitUntilAjaxIsLoaded()
-{
-    if($('#band_disco table a').length > 0)
+function generateTemplate(html, cpStr) {
+    let copiedTemplateString = `<div style="color:#FFF;background: #6D4C4C; padding: 3px; position: absolute; top: -20px; left: -20px; white-space: nowrap"></div>`.trim();
+    let copiedTemplate = document.createElement('template');
+    copiedTemplate.innerHTML = copiedTemplateString;
+    let template = document.createElement('template');
+    html = html.trim();
+    template.innerHTML = html;
+    let popUp = template.content.children[0];
+    template.content.children[1].addEventListener('click', (event) => {
+        event.preventDefault();
+        GM_setClipboard(cpStr + "\n");
+        popUp.innerText = `Copied "${cpStr}" to clipboard`;
+        popUp.style.position = 'relative';
+        setTimeout(() => {
+            popUp.style.display = 'none';
```

Comment:
This needs to be reset, otherwise each button will only show the notification once.

```javascript
popUp.style.display = 'block';
setTimeout(() => {
     popUp.style.display = 'none';
```

Reply:
Fixed in https://github.com/Row/metal-archives-userscripts/pull/3/commits/40f2814aae913ec035a08da220d4db34976d119d